fe/fe-core/src/main/java/org/apache/doris/analysis/CreateJobStmt.java

```diff
@@ -208,10 +210,24 @@ private void analyzerSqlStmt() throws UserException {
      * parse execute sql from create job stmt
      * Some stmt not implement toSql method,so we need to parse sql from originStmt
      */
-    private String parseExecuteSql(String sql) throws AnalysisException {
+    private static String parseExecuteSql(String sql, String jobName, String comment) throws AnalysisException {
         String lowerCaseSql = sql.toLowerCase();
-        int executeSqlIndex = lowerCaseSql.indexOf(" do ");
-        String executeSql = sql.substring(executeSqlIndex + 4).trim();
+        String lowerCaseJobName = jobName.toLowerCase();
```

Comment:
nit: maybe it is better to impl create job stmt in Nereids sometime

Reply:
yup. I'll start a thread later, but I think this might be in 2.2.x or 3.0